ensembl/datacheck/app.py

```diff
@@ -246,25 +247,41 @@ def job_list():
             job['status'] = 'failed'
 
     # if request.is_json:
-    return jsonify(jobs)
+    if format and format=='json': 
+        return jsonify(jobs)
+
+    return render_template('ensembl/datacheck/list.html')
     # else:
     # Need to pass some data to the template...
     # return render_template('ensembl/datacheck/list.html')
 
 
+@app.route('/datacheck/jobs/details', methods=['GET'])
+def job_details():
+
+  output_dir = request.args.get('output_dir')
+  db_name    = request.args.get('db_name')
+
+
+  file_data = open('/home/ensprod/vinay/web_datacheck/test/by_species.json', 'r').read()
```

Comment:
Seems to be a too much personal path

Reply:
code will be replaced with datcheck output path which is generated by pipeline

Reply:
This needs to point to the 'json_output_file' parameter, which is in the 'output' json of the completed datacheck job.